.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Feature Request
+about: Suggest a new feature for the rippled project
+title: "[Title with short description] (Version: [rippled version])"
+labels: Feature Request
+assignees: ''
+
+---
+<!-- Please search existing issues to avoid creating duplicates.-->
+
+## Description
```

Comment:
I would remove this heading, or change it to Summary with instructions to keep it short, i.e. a TLDR. They're going to elaborate the full description in the other sections.